Polyhedron/demo/Polyhedron/Plugins/Mesh_3/CMakeLists.txt

```diff
@@ -29,11 +29,17 @@ if (VTK_FOUND)
 else()
   message(STATUS "NOTICE : the DICOM files (.dcm) need VTK libraries to be open and will not be able to.")
 endif()
+
+find_package(Boost REQUIRED OPTIONAL_COMPONENTS filesystem)
```

Comment:
My error: the CMake module `FindBoost.cmake` only supports `OPTIONAL_COMPONENTS` since CMake-3.11.